Makefile

```diff
@@ -17,10 +16,10 @@ DOCKER_BUILD_OPTIONS =
 
 GIT_COMMIT=$(shell git rev-parse --short HEAD)
 
+export DOCKER_BUILDKIT = 1
+
 nginx-ingress:
```

Comment:
maybe we should rename `nginx-ingress` target to something that reflects that you're building the binary locally and remove `BUILD_IN_CONTAINER`

Reply:
Agreed. 

`make binary` makes sense to me